src/store/index.js

```diff
@@ -4,49 +4,32 @@
 
 import createStore from 'atom'
 import devtools from 'atom/devtools'
+import { requestsReducer, actions as reqActions } from '@app-elements/use-request'
 
 import { DEBUG } from '/consts'
-
-// Load state set by react-snap during prerendering
-const preloadedState = window.__PRELOADED_STATE__
-delete window.__PRELOADED_STATE__ // let it get garbage-collected
-
-// Define the global state on page load.
-export const initialState = {
-  clicks: 0,
-  token: window.localStorage.getItem('token'),
-  // In the browser, we initialize the currentPath prop, which is set
-  // by [Router](https://github.com/inputlogic/elements/tree/master/components/router)
-  currentPath: typeof window !== 'undefined'
```

Comment:
Looks like we lost `currentPath`?

Reply:
@weslord good catch. Updated to init that value, and keep it updated from the Router components.